Kernel/Arch/x86/PageDirectory.h

```diff
@@ -16,12 +17,11 @@ class PageTableEntry;
 
 class PageDirectoryEntry {
 public:
-    const PageTableEntry* page_table_base() const { return reinterpret_cast<PageTableEntry*>(m_raw & 0xfffff000u); }
-    PageTableEntry* page_table_base() { return reinterpret_cast<PageTableEntry*>(m_raw & 0xfffff000u); }
+    PhysicalPtr page_table_base() const { return PhysicalAddress::physical_page_base(m_raw); }
     void set_page_table_base(u32 value)
     {
         m_raw &= 0x8000000000000fffULL;
```

Comment:
Can you change that to use the same case through out, while you are at it,
it looks weird changing from lower to upper case

Reply:
I'm not sure what you're referring to? Are you by chance referring to the lines that were removed?